packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -263,15 +278,27 @@ export class LitContracts {
         throw new Error(msg);
       }
 
-      const chainInfo = {
+      const chronicleChainInfo = {
         chainId: '0x2AC49',
         chainName: 'Chronicle - Lit Protocol Testnet',
         nativeCurrency: { name: 'LIT', symbol: 'LIT', decimals: 18 },
         rpcUrls: this.rpcs,
-        blockExplorerUrls: [BLOCK_EXPLORER],
+        blockExplorerUrls: ['https://chain.litprotocol.com/'],
+        iconUrls: ['future'],
+      };
+
+      const vesuviusChainInfo = {
+        chainId: '0x907',
+        chainName: 'Vesuvius - Lit Protocol Devnet',
+        nativeCurrency: { name: 'testLit', symbol: 'testLit', decimals: 18 },
+        rpcUrls: this.rpcs,
+        blockExplorerUrls: ['https://vesuvius-explorer.litprotocol.com/'],
         iconUrls: ['future'],
       };
```

Comment:
This seems like a strange duplication of the source of truth for this data -- why can't we use our entries in `constants` for this?